docs/cliwrap.md

```diff
@@ -3,7 +3,13 @@ parent: Experimental features
 nav_order: 1
 ---
 
-# Wrapping other CLI entrypoints
+# DEPRECATED: Wrapping other CLI entrypoints
+
+**This functionality is now deprecated and slated for removal**. 
```

Comment:
If it is not known in what upcoming breaking-changes version it's going to be removed it's better to specify it.

Reply:
Totally agree, but I have no idea when we'll be able to do that unfortunately right now.

Given that we documented enabling cliwrap for kernel overrides today, it may be far into the future. Ideally though we'll have the kernel replacement replacement ready by the time this one actually ships in a release.